bndtools.core/bndtools.gtk.linux-eclipse-neon.x86_64.bndrun

```diff
@@ -0,0 +1,18 @@
+# use this runfile when you are on GTK 3.20+ 
+# because the version from Eclipse 4.4.2 doesnt work with newer versions of GTK
+
+-include: bndtools.shared.bndrun
+
+-runbundles: \
+	${crossplatform.bundles},\
+	org.eclipse.core.filesystem.linux.x86_64,\
+	org.eclipse.core.net.linux.x86_64,\
+	org.eclipse.swt.gtk.linux.x86_64
```

Comment:
Actually I had this a couple of minutes ago, but I thought I is easier for maintenance if we grab the latest available version in the repo.

I also think we can remove this duplicate dependency once BndTools 4 is upgraded to a recent Eclipse-base.

Nevertheless, I will update accordingly